plugins/ci/skills/prow-job-artifact-search/prow_job_artifact_search.py

```
@@ -0,0 +1,332 @@
+def cmd_fetch(prefix, filepath, max_bytes=DEFAULT_MAX_BYTES):
+    """Fetch contents of a specific file from GCS."""
+    target = gcs_path(prefix, filepath)
+
+    # Download to a temp file, then read
+    import tempfile
+
+    with tempfile.NamedTemporaryFile(delete=False, suffix=".artifact") as tmp:
+        tmp_path = tmp.name
+
+    try:
+        stdout, stderr, rc = run_gcloud(
```

Ruff (0.15.0)

[warning] Unpacked variable `stdout` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)